pyproject.toml

```diff
@@ -1,3 +1,78 @@
+[project]
+name = "tom-classifications"
+description = "Reusable TOMToolkit app to support classifying incoming targets."
+authors = [
+    {name = "TOM Toolkit Project", email = "tomtoolkit-maintainers@lco.global"}
+]
+maintainers = [
+    {name = "Brendan Mills"},
+    {name = "Joey Chatelain", email = "jchate6@gmail.com"},
+    {name = "William Lindstrom", email = "llindstrom@lco.global"},
+    {name = "Rachel Street", email = "rstreet@lco.global"},
+]
+license = {text = "GPL-3.0-only"}
+readme = "README.md"
+repository = "https://github.com/TOMToolkit/tom_registration"
+keywords = [
+    'tomtoolkit',
+    'TOM',
+    'astronomy',
+    'astrophysics',
+    'cosmology',
+    'science',
+]
+classifiers = [
+    'Environment :: Web Environment',
+    'Framework :: Django',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Topic :: Scientific/Engineering :: Astronomy',
+    'Topic :: Scientific/Engineering :: Physics'
+]
+dynamic = ["version"]
+requires-python = ">=3.9.0,<3.13"
+dependencies = [
+    "tomtoolkit (>=2.23.1,<3.0.0)"
+]
+
+[tool.poetry]
+version = "0.0.0" # version supplied by poetry-dynamic-versioning
+
+[tool.poetry.group.test.dependencies]
+factory_boy = ">3.2.1,<3.4"
+psycopg2-binary = "*" # for testing postgres
+
+[tool.poetry.group.coverage.dependencies]
+coverage = ">=6,<8" # coveralls needs ~6 even though 7.3.2 is latest
+coveralls = ">=3,<5"
+
+[tool.poetry.group.lint.dependencies]
+flake8 = ">=6.0,<7.2"
+
+[tool.poetry.requires-plugins]
+poetry-dynamic-versioning = { version = ">=1.0.0,<2.0.0", extras = ["plugin"] }
+
+[tool.poetry-dynamic-versioning]
+enable = true
+vcs = "git"
+style = "pep440"
+# the default pattern regex makes the 'v' manditory
+# this pattern modifies the default regex in order to make the 'v' optional
+# ('v' becomes '[v]?' meaning a single v, [v], and ? means optional)
+pattern = "(?x)^[v]?((?P<epoch>\\d+)!)?(?P<base>\\d+(\\.\\d+)*)([-._]?((?P<stage>[a-zA-Z]+)[-._]?(?P<revision>\\d+)?))?(\\+(?P<tagged_metadata>.+))?$"
+
+# substitute version not only in pyproject.toml (which the config block above does)
+# but also the __version__.py file (using the default value of the files property).
```

Comment:
Thanks for incorporating dynamic versioning. How about adding the `__version__.py` file so it has somewhere to write the version. (I'm less concerned about plumbing the version to the UI at this point, but that's probably a good idea also).

Reply:
made a new issue for this.